templates/etc/profile.d/tlog.csh.epp

```diff
@@ -33,11 +33,12 @@ if ( `getent passwd $TLOG_UID | cut -d':' -f7` != "/usr/bin/tlog-rec-session" )
   if ( -f "$TLOG_USERS" ) then
     if ( ! ($?TLOG_RUNNING) ) then
 
-      set TLOG_UNAME=`id -un`
-      set TLOG_GNAME=`id -gn`
+      set TLOG_UNAME=`id -nu`
```

Comment:
Although I see the acceptance test for bash that exercises the group logic for /bin/bash, I didn't see anything for csh.